packages/yourloops/test/unit/components/alarm/alarm-content-configuration-hook.test.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { renderHook } from '@testing-library/react-hooks'
+import useAlarmsContentConfiguration, {} from '../../../../components/alarm/alarms-content-configuration.hook'
+import { UnitsType } from '../../../../lib/units/models/enums/units-type.enum'
+import { buildTeam, createPatient } from '../../common/utils'
+
+const patient = createPatient()
+const teamId = 'teamId'
+const team = buildTeam(teamId)
+
+const monitoring = team.monitoring = {
+  enabled: true,
+  parameters: {
+    bgUnit: UnitsType.MGDL,
+    lowBg: 1,
+    highBg: 2,
+    outOfRangeThreshold: 10,
+    veryLowBg: 40,
+    hypoThreshold: 45,
+    nonDataTxThreshold: 50,
+    reportingPeriod: 7
+  }
+}
+describe('AlarmsContentConfiguration', () => {
```

Comment:
Missing tests:

- Testing the value of `thresholds`
- Testing the value of `saveButtonDisabled`
- Testing the method `resetToTeamDefaultValues`
- Testing the method `save`